frontend/src/components/worksheets/Worksheet.js

```diff
@@ -872,7 +872,7 @@ class Worksheet extends React.Component {
                                                             key={'title' + this.canEdit()}
                                                             canEdit={this.canEdit()}
                                                             fieldName='title'
-                                                            value={info && info.title}
+                                                            value={info && info.title || "untitled"}
```

Comment:
Let's do '(untitled)'.

Nit: we tend to use single quotes everywhere so try to be consistent (even though the codebase isn't quite consistent yet).

Reply:
I think "Untitled Worksheet" might look better, what do you think? (based off making "Untitled Document" in Microsoft Word)

Reply:
I think "Untitled Worksheet" might look better, what do you think? (based off making "Untitled Document" in Microsoft Word)

Reply:
The only issue is that our new front-end has a different set of naming systems with `untitled` and `unnamed`. I think for consistency we should make sure there's a unified naming system for both the old and new version (i.e., backward compatibility)? @percyliang what do you think?

Reply:
I don't feel strongly about this..."Untitled Worksheet" seems fine to me.  I guess the difference is that the worksheet title is actually not "Untitled Worksheet"...

Reply:
Oh, I see, since the worksheet title is actually not "Untitled Worksheet", then it makes sense to have a non-capitalized placeholder like "untitled".